ee/tabby-webserver/src/service/background_job/web_crawler.rs

```diff
@@ -19,9 +19,17 @@ impl WebCrawlerJob {
     }
 
     pub async fn run(self, embedding: Arc<dyn Embedding>) -> tabby_schema::Result<()> {
-        tabby_scheduler::crawl_index_docs(&self.source_id, &self.url, embedding, move |url| {
-            logkit::info!("Fetching {}", url);
-        })
+        tabby_scheduler::crawl_index_docs(
+            &self.source_id,
+            &self.url,
+            embedding,
+            move |url| {
```

Comment:
ditto - you can refactor this out and just invoke logkit::info directly, without this callback.